### ISSUE TITLE
IDebugProtocolServer.initialize has wrong return type

### ISSUE THREAD

**Issue by jonahkichwacoders** [eclipse/lsp4j#141]
`initialize`'s response body definition is a `$ref` (the only one currently used as a response body in the protocol definition). As a result the generator is creating the wrong definition:

This:
```java
CompletableFuture<InitializeResponse> initialize(InitializeRequestArguments args);
```
vs:
```java
CompletableFuture<Capabilities> initialize(InitializeRequestArguments args);
```

**Comment by spoenemann** [comment]
Any ideas how the generator could be integrated into this repo without IP problems? Do we need to reimplement it from scratch?

**Comment by jonahkichwacoders** [comment]
I am raising a new issue to track the generator IP/rewrite/other options.

**Comment by jonahkichwacoders** [comment]
Merged in PR  #148
